__tests__/app.test.js

```diff
@@ -378,3 +378,80 @@ describe('GET /api/articles', () => {
     })
 
 })
+
```

Comment:
tests below would need to reflect the requirement that the comment returned comes back on the key of 'comment'